bin/ghe-backup

```diff
@@ -79,7 +79,7 @@ if ! output=$(rsync -a src/ dest1 2>&1 && rsync -av src/ --link-dest=../dest1 de
   exit 1
 fi
 
-if [ "$(ls -il dest1/testfile | awk '{ print $1 }')" != "$(ls -il dest2/testfile | awk '{ print $1 }')" ]; then
+if [ "$(stat -c %i dest1/testfile)" != "$(stat -c %i dest2/testfile)" ]; then
```

Comment:
`stat -c %i` is much more efficient and simple here. As a side effect we lose the false positive `SC2012` error.